include/ftl/fiber.h

```diff
@@ -56,45 +55,41 @@ inline std::size_t RoundUp(std::size_t numToRound, std::size_t multiple);
 
 typedef void (*FiberStartRoutine)(void *arg);
 
-
 class Fiber {
 public:
 	/**
 	 * Default constructor
 	 * Nothing is allocated. This can be used as a thread fiber.
 	 */
-	Fiber()
-		: m_stack(nullptr),
-		  m_systemPageSize(0),
-		  m_stackSize(0),
-		  m_context(nullptr),
-		  m_arg(0) {
+	Fiber() : m_stack(nullptr), m_systemPageSize(0), m_stackSize(0), m_context(nullptr), m_arg(0) {
 	}
 	/**
 	 * Allocates a stack and sets it up to start executing 'startRoutine' when first switched to
 	 *
-	 * @param stackSize        The stack size for the fiber. If guard pages are being used, this will be rounded up to the next multiple of the system page size
```

Comment:
This change is incorrect. It should either be all on the same line, or wrap with indentation to match the `@param`

Reply:
This was an error in an old .clang-format file before I set `@param` as a "Comment Macro" so it wouldn't touch it. I fix it in a later commit.

Reply:
Sorry, I'm going commit by commit in order to have sane diffs.

Reply:
No worries, I meant to make a note about it, but I made the change yesterday and I forgot. 😄